chapters/introduction.tex

```diff
@@ -30,94 +39,97 @@ \section{Doing credible research at scale}
+Many researchers have been trained to code correctly. 
+However, when your code runs on your computer and you get the correct results, you are only half-done writing \underline{good} code.
+Good code is easy to read and replicate, making it easier to spot mistakes.
+Good code reduces noise due to sampling, randomization, and cleaning errors.
+Good code can easily be reviewed by others before it's published and replicated afterwards.
 
-Research reproduciblity and data quality follow naturally from
-good code and standardized practices.
 Process standardization means that there is
 little ambiguity about how something ought to be done,
-and therefore the tools that are used to do it are set in advance.
-Good code is easier to read and replicate, making it easier to spot mistakes.
-The resulting data contains substantially less noise
-that is due to sampling, randomization, and cleaning errors. 
-And all data work can be easily reviewed before it's published and replicated afterwards.
-
-A good do-file consists of code that has two elements:
-- it is correct (doesn't produce any errors along the way)
-- it is useful and comprehensible to someone who hasn't seen it before (such that the person who wrote this code isn't lost if they see it three weeks later)
-Most research assistants that join our unit have only been trained in how to code correctly.
-While correct results are extremely important, we usually tell our new research assistants that
-\textit{when your code runs on your computer and you get the correct results then you are only half-done writing \underline{good} code.}
-
-Just as data collection and management processes have become more social and collaborative,
-code processes have as well.\sidenote{\url{https://dimewiki.worldbank.org/wiki/Stata_Coding_Practices}} This means other people need to be able to read your code.
-Not only are things like documentation and commenting important,
-but code should be readable in the sense that others can:
+and therefore the tools to do it can be set in advance.
+Standard processes for code help other people to ready your code.\sidenote{
+\url{https://dimewiki.worldbank.org/wiki/Stata_Coding_Practices}} 
+Code should be well-documented, contain extensive comments, and be readable in the sense that others can:
 (1) quickly understand what a portion of code is supposed to be doing;
 (2) evaluate whether or not it does that thing correctly; and
 (3) modify it efficiently either to test alternative hypotheses
 or to adapt into their own work.\sidenote{\url{https://kbroman.org/Tools4RR/assets/lectures/07_clearcode.pdf}}
 
 To accomplish that, you should think of code in terms of three major elements:
 \textbf{structure}, \textbf{syntax}, and \textbf{style}.
-We always tell people to ``code as if a stranger would read it'',
-from tomorrow, that stranger will be you.
+We always tell people to "code as if a stranger would read it"
+(from tomorrow, that stranger will be you).
```

Comment:
```suggestion
(from tomorrow, the person most likely to be that stranger is you).
```

The way it read could be interpreted as the stranger was only you, when I think it should read more as if the stranger could just as well be you.

Reply:
revised in 66e59e0b09422f5de018fdb1a5cfedf1d995c34e